src/renderers/dom/shared/DOMProperty.js

```diff
@@ -208,22 +212,34 @@ var DOMProperty = {
   getPossibleStandardName: __DEV__ ? {autofocus: 'autoFocus'} : null,
 
   /**
-   * All of the isCustomAttribute() functions that have been injected.
+   * Checks whether a property name is a writeable attribute.
+   * @method
    */
-  _isCustomAttributeFunctions: [],
+  isWriteable: function(name, value) {
+    if (DOMProperty.isReservedProp(name)) {
+      return false;
+    }
+
+    if (value == null || DOMProperty.properties.hasOwnProperty(name)) {
+      return true;
+    }
+
+    let type = typeof value;
+
+    return type !== 'function' && type !== 'object';
```

Comment:
Good call. I don't know what to believe anymore now that `document.all` is in my life. Does this update (be4c9328cd5ec54bf9f132dc92214b47227ebf5f) make sense?